sdk/maps/Azure.Maps.Routing/src/MapsRoutingClient.cs

```diff
@@ -968,7 +968,10 @@ private static BatchRequest RouteDirectionsQueriesToBatchItems(IEnumerable<Route
                 }
                 if (options?.SectionFilter != null)
                 {
-                    uri.AppendQuery("sectionType", options.SectionFilter.Value.ToString(), true);
+                    foreach (var param in options.SectionFilter)
+                    {
+                        uri.AppendQuery("sectionType", param.ToString(), true);
```

Comment:
Is the intent here really to build a querystring with multiple duplicates of the same parameter name?

```text
http://some.url.com/thing?sectionType=1&sectionType=2&sectionType=3
```

Reply:
Yes, this is intended for V1 API. We've changed and avoid this pattern in our new V2 API.